README.md

```diff
@@ -2,7 +2,7 @@
 
 # Introduction
 
-This is the golang guest utilities for XenServer
+This is the golang guest utilities for {{PRODUCTNAME}}
```

Comment:
Not sure, but maybe change to "guest on Xen Hypervisor" is a better option

Reply:
as a variable name. what a mouthful^_^

Reply:
change to "Xenxerver VM Tool" on next/ "Citrix Hypervisor VM Tool" on Yangtze

Reply:
Hi @LunfanZhang , this should be changed to `This is {{PRODUCTNAME}} implemented with golang`